geo_autoRIFT/geogrid/src/geogrid.cpp

```diff
@@ -1270,12 +1311,13 @@ void geoGrid::geogrid()
                         raster2b[jj] = nodata_out;
                     }
                     
-                    for(int pp=0; pp<3; pp++)
-                    {
-                        targXYZ[pp] -= xyz[pp];
-                    }
+                    // range/azimuth pixel displacement to range/azimuth velocity
                     raster1c[jj] = dr/dt*365.0*24.0*3600.0*1;
-                    raster2c[jj] = norm_C(targXYZ)/dt*365.0*24.0*3600.0*1;
+                    raster2c[jj] = norm_C(da)/dt*365.0*24.0*3600.0*1;
```

Comment:
And why doe raster1c/2c get changed here as well... I ask because if there are changes to raster1a,b,c/2a,bc then we can't simply apply the scale factors (sf_raster1/2) to correct previously run data.

Reply:
@alex-s-gardner 
- raster1c and raster2c are range/azimuth pixel displacement to range/azimuth velocity conversion coefficients. Same reason as above: correction of pixel size misuse. 
- I understand your concern. But by any means, we have to rerun almost everything. For calculating the scale factors, we need transformed displacement vector in map projection CS (norm(xdiff/ydiff) for optical and norm(drpos/alt) for radar), by re-running Geogrid (which is also required to correct the pixel size misuse in conversion matrix calculations). Geogrid takes the ISCE files as input. Since all the ISCE files were removed after processing, we have to re-run ISCE. For autoRIFT, if @jhkennedy Joe saved that intermediate netcdf file "autoRIFT_intermediate.nc", we don't need to rerun autoRIFT but use that file for re-packaging; otherwise, autoRIFT needs to be re-ran as well.